spec/factories/preferences.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: preferences
+#
+#  id              :bigint           not null, primary key
+#  name            :string
+#  description     :text
+#  restriction     :boolean          default(FALSE)
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  user_id         :bigint           not null
+#  new_restriction :boolean          default(FALSE)
+#
+# Indexes
+#
+#  index_preferences_on_user_id  (user_id)
+#
+FactoryBot.define do
+  factory :preference do
+    name { "MyString" }
+    description { "MyText" }
```

Comment:
For future PRs you can check the use of [Faker gem](https://github.com/faker-ruby/faker). It is great for generating dummy/random data.

```suggestion
    name { Faker::Name.unique.name }
    description { Faker::Lorem.sentence }
```